src/main/scala/strawman/collection/SortedMap.scala

```diff
@@ -12,7 +12,9 @@ trait SortedMap[K, +V]
 
 trait SortedMapOps[K, +V, +CC[X, Y] <: SortedMap[X, Y] with SortedMapOps[X, Y, CC, _], +C <: SortedMap[K, V]]
   extends MapOps[K, V, Map, C]
-     with SortedOps[K, C] {
+     with SortedOps[K, C, SortedSet] {
+
+  def sortedIterableFactory = SortedSet
```

Comment:
Does that mean that when I navigate through the `sortedIterableFactory` of a `SortedMap` then I get back a `SortedSet`?